lib/components/ModalPortal.js

```diff
@@ -183,20 +183,20 @@ var ModalPortal = module.exports = React.createClass({
   },
 
   render: function() {
-    var contentStyles = (this.props.className) ? {} : this.props.defaultStyles.content;
-    var overlayStyles = (this.props.overlayClassName) ? {} : this.props.defaultStyles.overlay;
+    var defaultContentStyles = (this.props.className) ? {} : this.props.defaultStyles.content;
+    var defaultOverlayStyles = (this.props.overlayClassName) ? {} : this.props.defaultStyles.overlay;
```

Comment:
I mean...It's better to keep changes within the scope of the PR, because it may affect other PRs (and smaller diffs always wins). But if it does not conflict with other PRs, it's fine. :)